src/jquery.i18n.fallbacks.js

```diff
@@ -1,4 +1,4 @@
-/*!
+/* !
```

Comment:
jscs-jsdoc's [jsduck5 preset](//github.com/jscs-dev/jscs-jsdoc/blob/d1f3473723135018291ed17a2407f1cff25d0b96/lib/tags/jsduck5.json) does not accept `@licence` tags, therefore in https://github.com/wikimedia/jquery.i18n/pull/92#discussion-diff-57067989 it was decided to make jscs-jsdoc ignore those comments altogether instead of adding `@licence` as an extra tag.